packages/client/webapp/src/App.vue

```diff
@@ -33,5 +33,29 @@ const isAuthenticated = computed(() => auth.isAuthenticated);
 main {
 	display: flex;
 	flex-grow: 1;
+	isolation: isolate;
+}
+
+/* Sidebar */
+main > nav {
+	z-index: 2;
+}
+
+/* Pages */
+main > section {
+	z-index: 1;
+}
+</style>
+<style>
+body {
+	isolation: isolate;
+}
+
+header {
```

Comment:
Do we want these unscoped styles here? Maybe it would be better to apply a class to the `Header` component and style that within the `scoped` tag. Similarly, can we move the `main` styles below up to the scoped section?

Reply:
I am not a fan of adding extra attributes when unnecessary, but I guess that's part of the single web app design.